docs/source/usage_guides/distributed_inference.md

```diff
@@ -216,13 +216,10 @@ with torch.no_grad():
     output = model(*args)
 ```
 
-When finished, all the data will be on the last GPU, which you can use the [`PartialState`] to find and extract:
+When finished, all the data will be on the CPU on each node:
 
```

Comment:
we might need to update this

Reply:
Will be updated in the prior PR, I'll resolve the conflict after it's merged (if there is one :) )